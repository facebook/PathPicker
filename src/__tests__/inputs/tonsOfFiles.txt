
./tests/tests.c
./arch/x86/kernel/cpu.c
./arch/x86/kernel/ioapic.c
./arch/x86/kernel/apic.c
./arch/x86/kernel/gdt.c
./arch/x86/kernel/idt.c
./drivers/timer/i8254.c
./drivers/input/kbd.c
./drivers/acpi/acpica/rsinfo.c
./drivers/acpi/acpica/psxface.c
./drivers/acpi/acpica/evmisc.c
./drivers/acpi/acpica/nseval.c
./drivers/acpi/acpica/hwsleep.c
./drivers/acpi/acpica/nsprepkg.c
./drivers/acpi/acpica/nsaccess.c
./drivers/acpi/acpica/tbutils.c
./drivers/acpi/acpica/uteval.c
./drivers/acpi/acpica/psscope.c
./drivers/acpi/acpica/dsinit.c
./drivers/acpi/acpica/nswalk.c
./drivers/acpi/acpica/utxferror.c
./drivers/acpi/acpica/utascii.c
./drivers/acpi/acpica/tbinstal.c
./drivers/acpi/acpica/utprint.c
./drivers/acpi/acpica/evevent.c
./drivers/acpi/acpica/dsdebug.c
./drivers/acpi/acpica/psloop.c
./drivers/acpi/acpica/utstrtoul64.c
./drivers/acpi/acpica/utaddress.c
./drivers/acpi/acpica/evgpeutil.c
./drivers/acpi/acpica/dsobject.c
./drivers/acpi/acpica/exregion.c
./drivers/acpi/acpica/nsload.c
./drivers/acpi/acpica/utnonansi.c
./drivers/acpi/acpica/utxfinit.c
./drivers/acpi/acpica/evxface.c
./drivers/acpi/acpica/exconvrt.c
./drivers/acpi/acpica/hwtimer.c
./drivers/acpi/acpica/utdebug.c
./drivers/acpi/acpica/exoparg1.c
./drivers/acpi/acpica/utexcep.c
./drivers/acpi/acpica/nsxfobj.c
./drivers/acpi/acpica/hwgpe.c

