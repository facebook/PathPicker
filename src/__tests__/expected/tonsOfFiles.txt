./drivers/acpi/acpica/utxferror.c
./drivers/acpi/acpica/utascii.c
./drivers/acpi/acpica/tbinstal.c
./drivers/acpi/acpica/utprint.c
./drivers/acpi/acpica/evevent.c
./drivers/acpi/acpica/dsdebug.c
./drivers/acpi/acpica/psloop.c
./drivers/acpi/acpica/utstrtoul64.c
./drivers/acpi/acpica/utaddress.c
./drivers/acpi/acpica/evgpeutil.c
./drivers/acpi/acpica/dsobject.c
./drivers/acpi/acpica/exregion.c
./drivers/acpi/acpica/nsload.c
./drivers/acpi/acpica/utnonansi.c
./drivers/acpi/acpica/utxfinit.c
./drivers/acpi/acpica/evxface.c
./drivers/acpi/acpica/exconvrt.c
./drivers/acpi/acpica/hwtimer.c
./drivers/acpi/acpica/utdebug.c
./drivers/acpi/acpica/exoparg1.c
./drivers/acpi/acpica/utexcep.c
./drivers/acpi/acpica/nsxfobj.c
./drivers/acpi/acpica/hwgpe.c
==========================================================
Type a command below! Paths will be appended or replace $F
Enter a blank line to go back to the selection process
==========================================================
..........................................................
________________________________________________________________________________
command examples: | git add | git checkout HEAD~1 -- | mv $F ../here/ |